core/gather/base-gatherer.js

```diff
@@ -55,52 +54,6 @@ class FRGatherer {
    * @return {LH.Gatherer.PhaseResult}
    */
   getArtifact(passContext) { }
-
-  /**
-   * Legacy property used to define the artifact ID. In Fraggle Rock, the artifact ID lives on the config.
-   * @return {keyof LH.GathererArtifacts}
-   */
-  get name() {
```

Comment:
Could have kept this for convenience, but sine it requires some `substr` hackery just to work with rollup I figured it's best not to rely on it if we don't have to.